content/integrate/write-behind/data-transformation/data-transformation-pipeline.md

```diff
@@ -38,12 +38,11 @@ default.yaml
 ```yaml
 source:
   table: "*"
-  row_format: full
```

Comment:
I'm not sure if line 53 will still work if we remove it here, we probably should come up with a better example.
BTW, it also seems that this example is scaffolded in `default.yaml.example`, not sure if this ideal - @yaronp68